bin/rl/rl.cpp

```diff
@@ -4167,8 +4170,8 @@ ProcessConfig
 
       // Walk the condition nodes looking for applicability.
 
-      ConditionNodeList * conditionNodeList = NULL;
-      ConditionNodeList * conditionNodeLast = NULL;
+      conditionNodeList = NULL;
+      conditionNodeLast = NULL;
```

Comment:
The 'goto Label_Skip' a few lines above will skip the initialization, which is not allowed by the C++ standard.  There are a few options of ways to fix this.  I can remove the redundant "=NULL" at the old location, or I can move the initialization to a separate line and keep it in the original location, which would look like this:

ConditionNodeList * conditionNodeLast;
conditionNodeLast = NULL